tables-example/src/app/extensions/TableExampleCard.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import {
+  Table,
+  TableHead,
+  TableRow,
+  TableHeader,
+  TableBody,
+  TableCell,
+  Input,
+  Flex,
+  Heading,
+  Text,
+  hubspot,
+} from '@hubspot/ui-extensions';
+
+// Define the extension to be run within the Hubspot CRM
+hubspot.extend(() => <TableExampleCard />);
+
+const tableData = [
+  {
+    name: 'Instapage',
+    publishStatus: 'Published',
+    review: '-',
+    installs: '-',
+    lastUpdated: 'March 8, 2019 (Amelia Merchant)',
+    createdAt: 'March 8, 2019 (Amelia Merchant)',
+  },
+  {
+    name: 'Spotify',
+    publishStatus: 'Draft',
+    review: '41',
+    installs: '41',
+    lastUpdated: 'March 8, 2019 (Amelia Merchant)',
+    createdAt: 'March 8, 2019 (Amelia Merchant)',
+  },
+  {
+    name: 'Slack',
+    publishStatus: 'Published | Draft',
+    review: '136',
+    installs: '136',
+    lastUpdated: 'March 8, 2019 (Amelia Merchant)',
+    createdAt: 'March 8, 2019 (Amelia Merchant)',
+  },
+  {
+    name: 'Survey Monkey',
+    publishStatus: 'Published',
+    review: '136',
+    installs: '136',
+    lastUpdated: 'March 8, 2019 (Amelia Merchant)',
+    createdAt: 'March 8, 2019 (Amelia Merchant)',
+  },
+];
+
+const TableExampleCard = () => {
+  return (
+    <>
+      <Flex direction="column" gap="small">
+        <Heading>Table Example</Heading>
+        <Text>
+          Tables can be tricky, especially as they grow in complexity and size.
+          This is an example of how input, buttons, and scrolling can be
+          implemented.
+        </Text>
+
+        <Flex direction="row">
+          <Input name="search" placeholder="Search" />
+        </Flex>
+
+        <Table
+          bordered={true}
+          paginated={true}
```

Comment:
I think `paginated`, `pageCount`, and `showButtonLabels` should be removed now that we are not doing search and there are just five entries. In other words:  there's no need to paginate if everything fits in one page, let's simplify the table.